kahuna/public/js/upload/jobs/upload-jobs.js

```diff
@@ -79,6 +79,7 @@ jobs.controller('UploadJobsCtrl', [
 
                         imageService(image).states.canDelete.then(deletable => {
                             jobItem.canBeDeleted = deletable;
+                             if (image.data.softDeletedMetadata !== undefined) { ctrl.isDeleted = true; }
```

Comment:
I think this should set `jobItem.isDeleted` instead (and update the condition in upload-jobs.html to match) - it's possible for a user to upload both deleted and not-deleted images in the same batch